batch/requirements.txt

```
@@ -1,2 +1,2 @@
 google-cloud-batch==0.17.31
-google-cloud-logging==3.5.0
+google-cloud-logging==3.11.4
```

```suggestion
google-cloud-logging==3.5.0
```

remove due to unrelated failures blocking all other bumps